### PR TITLE
Updates to travis env

### PR DESCRIPTION
**Pull request by jklenzing**
Looking over the logs, the last passing Travis-CI test was over 8 months ago.  Tests running 7 months ago began failing.  This PR is intended to fix that.

Based on problems identified at pysat/pysatNASA#25

### CODE REVIEW

**Comment by jklenzing** [comment]
Noting that tests pass locally using pytest.  The problem is on the fortran integration at travis.

**Comment by jklenzing** [comment]
Note to self: turned off auto-build of PRs at Travis to cut down on testing time.  Only push is currently built

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,13 @@
 language: python
-jobs:
+dist: xenial
+matrix:
```
Comment (aburrell) [inline .travis.yml:3]: Isn't `jobs` preferred to `matrix`?

File: .travis.yml [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
   include:
-    - python: 2.7
     - python: 3.6
     - python: 3.7
```
Comment (aburrell) [inline .travis.yml:6]: The python versions should be put in double quotes ("3.6").

File: pysatCDF/tests/test_cdf.py [hunk 3]
```diff
@@ -1,27 +1,23 @@
...
         """Runs before every method to create a clean testing setup."""
-        # self.meta = pysat.Meta()
-        # self.testInst = pysat.Instrument('pysat', 'testing', tag='', clean_level='clean')
 
     def teardown(self):
         """Runs after every method to clean up previous testing."""
-        # del self.testInst
 
     def test_vefi_load(self):
         import os
```
Comment (aburrell) [inline pysatCDF/tests/test_cdf.py:14]: Probabaly want to clean this up by moving imports to top.  Might make debugging easier and could prevent double-import issues.

**Review by aburrell — COMMENTED** [review]
Some suggestions.  Nothing groundbreaking, unfortunately.

**Comment by jklenzing** [comment]
Since the code is being deprecated, we have two options outside of fixing this here:
- Upload a version of the develop-3 branch to test.pypi, install from there where needed
- Push the cdflib integration to be fixed ASAP so that we can drop this requirement

Thoughts?  @rstoneback, @aburrell, @JonathonMSmith

**Pull request closed without merging**